src/Resources/contao/widgets/FileTree.php

```diff
@@ -97,9 +97,9 @@ protected function validator($varInput)
 		}
 
 		// Store the order value
-		if ($this->orderField != '')
+		if ($this->orderField != '' && ($order = \Input::post($this->strOrderName)) !='')
```

Comment:
Wouldn’t this make it impossible to deselect files if the order field in the database has already some values stored?

Reply:
No. The order values will stay in the database but this does not prevent saving an empty selection itself.

When the files are selected again the saved order is applied directly. This could be a feature ;-)

Reply:
But we can change this to save empty order values as well:

```php
		// Store the order value
		if ($this->orderField != '')
		{
			$arrNew = !empty($order = \Input::post($this->strOrderName)) ? explode(',', $order) : array();

			// Only proceed if the value has changed
			if ($arrNew !== $this->{$this->orderField})
			{
				$this->Database->prepare("UPDATE {$this->strTable} SET tstamp=?, {$this->orderField}=? WHERE id=?")
							   ->execute(time(), serialize($arrNew), $this->activeRecord->id);

			    $this->objDca->createNewVersion = true; // see #6285
			}
		}
```

Reply:
Yes, I think saving empty order values would be better.

Reply:
I am of the same opinion and changed the PR accordingly.